Storage/ThirdParty/SwiftData.swift

```diff
@@ -89,6 +89,9 @@ public class SwiftData {
      * close a database connection and run a block of code inside it.
      */
     public func withConnection(flags: SwiftData.Flags, cb: (db: SQLiteDBConnection) -> NSError?) -> NSError? {
+#if TARGET_IS_EXTENSION
+        let targetId = UIApplication.sharedApplication().beginBackgroundTaskWithExpirationHandler { _ in }
+#endif
```

Comment:
`sharedApplication()` is not available in extensions. Did you mean `!TARGET_IS_EXTENSION` ?